assets/js/event.js

```diff
@@ -0,0 +1,165 @@
+$(document).ready(function() {
+AOS.init();
+const eventswiper = document.querySelector(".swiper-wrapper");
+const event = [
+    {
+        title : "Slide One",
+        image : "assets/images/event/image3.jpg",
+        text  : "Lorem ipsum, dolor sit amet consectetur adipisicing elit. Possimus optio odit exercitationem, minima non quisquam quibusdam similique error consequatur modi!"
```

Comment:
Avoid Lorem Ipsum. Put up text like "Talk on XYZ", "Keynote on ABC", "Group Picture"

Only the title is sufficient, Please have a look at the reference